### PR TITLE
Fix author chips wrapping

### PR DESCRIPTION
**Pull request by D8H**
- https://gdevelop-storybook.s3.amazonaws.com/fix-author-wrapping/latest/index.html?path=/story/assetstore-examplestore--default
- https://gdevelop-storybook.s3.amazonaws.com/fix-author-wrapping/latest/index.html?path=/story/assetstore-extensionstore--default

### CODE REVIEW

File: newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js [hunk 1]
```diff
@@ -182,14 +182,16 @@ const ExtensionInstallDialog = ({
...
             <Line>
-              {extensionShortHeader.authors &&
-                extensionShortHeader.authors.map(author => (
-                  <UserPublicProfileChip
-                    user={author}
-                    key={author.id}
-                    isClickable
-                  />
-                ))}
+              <div style={{ flexWrap: 'wrap' }}>
```
Comment (ClementPasteau) [inline newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js:185]: should this be inside the UserPublicProfileChip? 
I'm afraid that next time we use it we'll forget that wrapper

Comment (AlexandreSi) [reply newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js:185]: Or a whole component?

Comment (D8H) [reply newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js:185]: I was thinking of adding difficulty and size as chip on the begging of this line to save vertical space.

**Review by ClementPasteau — APPROVED** [review]
May be overkill to create a component for a flex wrap, so ok for now 👍

**Pull request merged**